File: MovieApi/ClientApp/src/Movie.ts
import { DecimalPipe } from "@angular/common";

export interface Movie {
  title: string;
  release_Date: string;
  overview: string | undefined;
  popularity: DecimalPipe;
  vote_Count: number;
  vote_Average: DecimalPipe
  original_Language: string
  genre: string
  poster_Url: string
}

File: MovieApi.Data/Models/Movie.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MovieApi.Data.Models
{
    public class Movie
    {
        public string Title { get; set; } = string.Empty;
        public DateTime Release_Date { get; set; }  
        public string Overview { get; set; } = string.Empty;
        public decimal Popularity { get; set; }
        public int Vote_Count { get; set; }
        public decimal Vote_Average { get; set; }
        public string Original_Language { get; set; } = string.Empty;
        public string Genre { get; set; } = string.Empty;
        public string Poster_Url { get; set; } = string.Empty;
    }
}
